cmake/CeleritasUtils.cmake

```diff
@@ -237,7 +254,49 @@ function(celeritas_add_library target)
   endif()
 
   # Build all targets in lib/
-  set_target_properties(${_targets} PROPERTIES ${_props}
+  set_target_properties(${_targets} PROPERTIES
+    ARCHIVE_OUTPUT_DIRECTORY "${CELERITAS_LIBRARY_OUTPUT_DIRECTORY}"
+    LIBRARY_OUTPUT_DIRECTORY "${CELERITAS_LIBRARY_OUTPUT_DIRECTORY}"
+  )
+
+  # Install all targets to lib/
+  install(TARGETS ${_targets}
+    EXPORT celeritas-targets
+    ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+    LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+    COMPONENT runtime
+  )
+endfunction()
+
+#-----------------------------------------------------------------------------#
+
+function(celeritas_add_library target)
```

Comment:
This seems to be the same line as line 231 (i.e. the function seems defined twice).  What am I missing?

Reply:
Whoops! Thanks @pcanal , it was a copy-paste error.